tests/modules/programs/firefox/profile-settings.nix

```diff
@@ -154,12 +157,18 @@ in {
       id = 5;
       containers = {
         "shopping" = {
-          id = 6;
```

Comment:
Wait what was wrong with the containers test? Each container should have its own unique id set, so I think it makes sense to keep one explicitly set in the test.

Reply:
It was a bit confusing, because it's container id, not profile id. It looks like someone set it to 6 because it's the next number after 5, but these ids are unrelated. I feel like there's no need to set container id here (apart from making this test failproof against changing the default container id), and it's not testing anything.

I could set it explicitly to 1, but using 6 here is a bit confusing, because it looks like it's related to this 5.